scrapers/HelixStudios.yml

```diff
@@ -0,0 +1,79 @@
+name: "HelixStudios"
+sceneByURL:
+  - action: scrapeXPath
+    url:
+      - https://www.helixstudios.com/video/
+    scraper: sceneScraper
+
+performerByURL:
+  - action: scrapeXPath
+    url:
+      - https://www.helixstudios.com/model/
+    scraper: performerScraper
+
+xPathScrapers:
+  sceneScraper:
+    common:
+      $info: //div[@class="text"]
+    scene:
+      Title: //title/text()
+      Date:
+        selector: //div[@class="info-items"]/span[@class="info-item date"]/text()
+        postProcess:
+          - replace:
+              - regex: "th|st|nd|rd"
+                with:
+          - parseDate: Jan 2, 2006
```

Comment:
I just noticed a couple more cases for the date.
`today` and `yesterday ` are covered already as are dates older than 2021
we need to account for recent videos that use `xx days ago` and for scenes that are in 2021 and so the year isnt shown

```yaml
        postProcess:
          - replace:
              - regex: "th|st|nd|rd"
                with:
              - regex: '(\d+)\sdays\sago'
                with: $1
              - regex: '^([a-zA-Z]+)\s(\d+)$'
                with: "$1 $2, 2021"
          - subtractDays: true
          - parseDate: Jan 2, 2006
```
seems to take everything into account
NOTE: for `subtractDays` a recent stash version is needed `stash >= v0.7.0-12`